app/views/pages/privacy.html.slim

```diff
@@ -0,0 +1,87 @@
+
+p If you apply for an RGSoC scholarship, we may also collect information such as your geolocation (only city), your age range, financial information (how much of a stipend you require monthly) and any diversity information that may need to be taken into account and that you wish to disclose with us (such as gender, sexual identity, health information). This data is used to rate applications and adapt the stipend to your requirements.
+br If you are a member of the selection committee, we collect your comments and ratings to the applications in order to process and evaluate applications.
+br If you are accepted into the RGSoC program as a participant, we may also collect your full name and address, as well as your T-Shirt size (required to send sponsor swag packages to you). 
+br If you are a supervisor or an organiser, we may also collect your full name and address, as well as your T-Shirt size, for sponsor swag package purposes.
+br Your sensitive information is only used internally and won't be shared with others.
+
+p The RGSoC Teams App also collects aggregated, anonymous user data regarding app usage. The user data we collect is used to improve our platform and the quality of our service. We only collect personal data that is required to provide our services, and we only store it insofar that it is necessary to deliver these services.
+
+
+h2 Your data
+The RGSoC Teams App uses third-party vendors and hosting partners to provide the necessary hardware, software, networking, storage, and related technology required to run the Teams App, such as Heroku for storing your data and GitHub for the purpose of further developing our RGSoC Teams App. Although RGSoC owns the databases for the RGSoC Teams application, you retain all rights to your data. We will never share your personal data with a 3rd party without your prior authorization, and we will never sell data to third-parties. We transfer data with third-parties necessary to our ability to provide our services, all of whom are GDPR-compliant and provide the necessary safeguards required if they are outside of the EU.
+
+
+h2 Your rights
+p For further information on your rights, to withdraw consent, or to ask for deletion of your data, you can contact us at <a href="mailto:contact@rgsoc.org">contact@rgsoc.org</a>.
```

Comment:
Probably personal preference, but using HTML within `slim` always looks a bit dangerous to me... 😉 

How about just using Rail's [mail_to](http://api.rubyonrails.org/v5.2.0/classes/ActionView/Helpers/UrlHelper.html#method-i-mail_to) helper?

Could look like this:
```slim
p For further information on your rights, to withdraw consent, or to ask for deletion of your data, you can contact us at #{mail_to('contact@rgsoc.org'}
```